src/installer.js

```diff
@@ -3,17 +3,26 @@ var fs = require("fs");
 var path = require("path");
 
 module.exports.check = function(dependencies, dirs) {
-  return dependencies.filter(function(dependency) {
+  var missing = [];
+  dependencies.forEach(function(dependency) {
     // Ignore relative modules, which aren't installed by NPM
-    if (!/^[a-z\-0-9]+$/.test(dependency)) {
-      return false;
+    if (/^\./.test(dependency)) {
```

Comment:
Oh yea, this is much simpler :) Truth be told, I used this regex from one of the Webpack configuration examples for `target: "node"`.